.github/CODEOWNERS

```diff
@@ -2,14 +2,14 @@
 # repo and would like to avoid introducing dependencies in the meantime.
 /builtins-default-costs/ @anza-xyz/svm
 /compute-budget/ @anza-xyz/svm
+/compute-budget-instruction/ @anza-xyz/svm
 /fee/ @anza-xyz/svm
 /log-collector/ @anza-xyz/svm
 /program-runtime/ @anza-xyz/svm
 /programs/bpf_loader/ @anza-xyz/svm
 /programs/compute-budget/ @anza-xyz/svm
 /programs/loader-v4/ @anza-xyz/svm
 /programs/system/ @anza-xyz/svm
-/runtime-transaction/ @anza-xyz/svm
```

Comment:
Can we remove transaction-view below now too?
My understanding is that it was only in this list because of runtime-transaction.

Reply:
It's now needed only due to this `agave_transaction_view::static_account_keys_frame::MAX_STATIC_ACCOUNTS_PER_PACKET` in `builtin_program_filters.rs`. Maybe we can move the constant to SDK. That'll remove the dep.

Reply:
Yeah, can probably go into SDK. It's just the max packet size divided by Pubkey size.

Thanks for clarifying

Reply:
please refer to pinned message in the SDK channel on Discord: 
![image](https://github.com/user-attachments/assets/e4b7fedc-5116-479f-a401-f243b729616d)

Reply:
Redefined the variable within the crate, and removed the dep.